randovania/games/dread/json_data/Burenia.txt

```diff
@@ -2594,7 +2594,7 @@ Extra - asset_id: collision_camera_021
   > Start Point 2
       Any of the following:
           Space Jump
-          Infinite Bomb Jump (Intermediate) and Lay Bomb
+          Infinite Bomb Jump (Advanced) and Lay Bomb
```

Comment:
Intermediate for Cross, advanced for normal

Reply:
Done